sunpy/visualization/animator/mapsequenceanimator.py

```diff
@@ -73,10 +73,7 @@ def updatefig(self, val, im, slider):
         i = int(val)
         im.set_array(self.data[i].data)
         im.set_cmap(self.mapsequence[i].plot_settings['cmap'])
-
         norm = deepcopy(self.mapsequence[i].plot_settings['norm'])
-        # The following explicit call is for bugged versions of Astropy's ImageNormalize
-        norm.autoscale_None(self.data[i].data)
```

Comment:
Ha, *that's* the problem?  I fixed the referenced Astropy bug in v1.0.5 (astropy/astropy#4117), so this workaround is ancient (2015, see #1542).

Reply:
I'll note that the fix I added to Astropy has since gone away, so presumably the implementation was improved and/or there have been changes in matplotlib.  2015 was a long time ago.